README.md

```diff
@@ -34,23 +34,23 @@ Purge [Kinsta](https://bit.ly/2NWj3sg) cache when [Trellis](https://github.com/r
 
 ## Installation
 
-Add this role to `requirements.yml`:
+Add this role to `galaxy.yml`:
 ```yaml
-# requirements.yml
+# galaxy.yml
 - src: https://github.com/ItinerisLtd/trellis-purge-kinsta-cache-during-deploy
   version: 0.3.0 # Check for latest version!
 ```
 
 Run the command:
 ```bash
-➜ ansible-galaxy install -r requirements.yml --force
+➜ ansible-galaxy install -r galaxy.yml --force
 ```
 
 ## Role Variables
 
 Add this role to the [`deploy_after` hook](https://roots.io/trellis/docs/deploys/#hooks):
 ```yaml
-# group_vars/all/deploy-hooks.yml
+# roles/deploy/defaults/main.yml
```

Comment:
This should be reverted.

`group_vars/all/deploy-hooks.yml` is where Trellis expects us to add custom config.

Reply:
Sure, I was afraid it was wrong...
Is the group_vars/all/deploy-hooks.yml file automatically used? 
What about the trellis/deploy-hooks folder? 
I just want to figure out the best way to use those hooks, but maybe I should ask this on discourse 🤓

Reply:
I read carefully the docs and you are absolutely right. My fault...